pandapower/plotting/plotly/simple_plotly.py

```diff
@@ -168,7 +169,9 @@ def simple_plotly(net, respect_switches=True, use_line_geodata=None, on_map=Fals
                                               hoverinfo_func=get_hoverinfo,
                                               filename=filename,
                                               auto_open=auto_open,
-                                              showlegend=showlegend)
+                                              showlegend=showlegend,
+                                              zoomlevel=zoomlevel,
```

Comment:
@jkisse you may add `zoomlevel` to the docstring of the function `simple_plotly()`